beforepush.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+if ! command -v shellcheck; then
+	echo "shellcheck not found on this machine. Please install from https://www.shellcheck.net/"
```

Comment:
You might even be more specific and give them this address too `https://github.com/koalaman/shellcheck#installing`